src/main/java/edu/nextstep/racingcar/step3/app/Car.java

```diff
@@ -0,0 +1,31 @@
+package edu.nextstep.racingcar.step3.app;
+
+public class Car {
+
+    private final MoveStrategy moveStrategy;
+    private int distance = 1;
+
+    public Car(ParamsValidator paramsValidator, MoveStrategy moveStrategy) {
+        paramsValidator.validate();
+
+        this.moveStrategy = moveStrategy;
+    }
+
+    public void move() {
+        if (moveStrategy.isMove()) {
+            this.distance++;
+        }
+    }
+
+    public int getDistance() {
+        return distance;
+    }
+
+    public interface ParamsValidator {
+        void validate();
+    }
+
+    public interface MoveStrategy {
+        Boolean isMove();
+    }
```

Comment:
```suggestion
    public interface MoveStrategy {
        boolean isMove();
    }
```
특별한 경우가 아니면 원시타입을 사용해주세요

Reply:
이 인터페이스도 외부에 정의해주면 더 좋을 것 같아요